tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/modules/HostOSGiBundleDefaultValueService.java

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+
+package com.liferay.ide.gradle.core.modules;
+
+import com.liferay.ide.core.util.CoreUtil;
+import com.liferay.ide.core.util.FileUtil;
+
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.sapphire.DefaultValueService;
+
+/**
+ * @author Terry Jia
+ */
+public class HostOSGiBundleDefaultValueService extends DefaultValueService
+{
+
+    @Override
+    protected String compute()
+    {
+        String projectName = op().getProjectName().content();
```

Comment:
This code like very similar to the code in NewModuleFragmentFilesOpMethods, they should be merged.